Jenkinsfile

```diff
@@ -67,81 +67,121 @@ pipeline {
                         */
                         echo "Not linting due to rpmlint not recnognizing Supplements:"
                     }
-                }
+                } // stage('RPM Lint')
+                stage('Check Packaging') {
```

Comment:
For consistency at least, these should be run in parallel.

But shouldn't we be using the pipeline-lib function for this?  It would just need a small change to allow the rpmlint stage to be skipped.  This might not be the last RPM we need to do that with.  Seems worthwhile to add.

Reply:
We have to suppress failing on rpm lint for the SLURM project now.
Both special cases for this so far are for SLES-12/LEAP-42 which is going to go away.